homeassistant/components/camera/synology.py

```diff
@@ -164,10 +169,12 @@ def get_session_id(hass, username, password, login_url, valid_cert):
     }
     try:
         with async_timeout.timeout(TIMEOUT, loop=hass.loop):
-            auth_req = yield from hass.websession.get(
+            auth_req = yield from aiohttp.get(

```

Comment:
`ClientSession` parse `get` to `request` and that support cookies. That is the same stuff with `auth`, you can set it on `ClientSession` global for all requests or selected only on `get/post/... -> request`.